src/main/kotlin/no/nav/familie/ef/sak/service/PersisterGrunnlagsdataService.kt

```diff
@@ -1,139 +0,0 @@
-package no.nav.familie.ef.sak.service
```

Comment:
Dette er flyttet til `GrunnlagsdataService.kt` 
Pga endringer i filene og rename så blir diffen som den er nå. Det jeg kan gjøre er å splitte opp dette i 2-3 ulike PR's. Sånn att denne bare endrer inneholdet, og sen blir rename av klasser i en egen for å beholde historikken i hver fil. (men blir fortsatt litt rart når denne blir registergrunnlagservice og registergrunnlagservice endrer navn til noe annet >.<)